R/utils.R

```diff
@@ -12,3 +12,51 @@ get_vrd_api <- function() {
     }
     x
 }
+
+
+#' call_vrd_api()
+#'
+#'helper function for querying api database
+#'
+#' @return vrd_api class
+#' @export
+#'
+#' @examples
+#' call_vrd_api()()
+
+call_vrd_api <- function(url_, query, limit=25, page=1){
+
```

Comment:
need to determine what we're doing with pagination and limits

Reply:
From the api's website, limit=25 and page=1 by default.

I think it makes most sense to include this parameters with each api call function.

Reply:
agreed